src/libmongoc/src/mongoc/mongoc-structured-log.c

```diff
@@ -1022,10 +1040,9 @@ _mongoc_structured_log_append_server_description (bson_t *bson,
    return stage + 1;
 }
 
-const mongoc_structured_log_builder_stage_t *
-_mongoc_structured_log_append_topology_as_description_json (bson_t *bson,
-                                                            const mongoc_structured_log_builder_stage_t *stage,
-                                                            const mongoc_structured_log_opts_t *opts)
+static mcommon_string_t *
+_mongoc_structured_log_topology_description_as_json (const mongoc_topology_description_t *td,
```

Comment:
```suggestion
_mongoc_structured_log_topology_description_to_json (const mongoc_topology_description_t *td,
```
Perhaps "description_to_json" may help disambiguate these helper functions a bit better from their "as_description_json" counterparts.